src/pages/new-task.vue

```diff
@@ -0,0 +1,38 @@
+<template>
+  <div>
+    <h1>Create new task</h1>
+    <form @submit.prevent>
+      <div class="form-group">
+        <label>Title:</label>
+        <input v-model="card.title" class="form-control" type="text">
+      </div>
+      <div class="form-group">
+        <label>Description:</label>
+        <textarea v-model="card.descr" class="form-control"></textarea>
+      </div>
+      <button :disabled="!card.title || !card.descr" @click="addNew" class="btn btn-primary">Submit</button>
```

Comment:
Actually it's much better to listen and process submit event on form than click on submit (e.g. customer can use keyboard for submit)